src/Adapter/Category/CommandHandler/AbstractDeleteCategoryHandler.php

```diff
@@ -35,23 +35,35 @@
  */
 abstract class AbstractDeleteCategoryHandler
 {
+    /**
+     * Get products that belong to a category
+     *
+     * @param int $categoryId
+     */
+    protected function getProductsWithoutCategory($categoryId)
```

Comment:
Products without category or category without products ?
I feel like the name of the function doesn't really match what it does